SPECS/thrift/thrift.spec

```diff
@@ -0,0 +1,691 @@
+%global php_extdir  %(php-config --extension-dir 2>/dev/null || echo "undefined")
+ 
+%{?perl_default_filter}
+%global __provides_exclude_from ^(%{python3_sitearch}/.*\\.so|%{php_extdir}/.*\\.so)$
+ 
+%global have_mongrel 0
+ 
+# erlang-jsx is available in F19 but orphaned in F22
+%global have_jsx 0
+ 
+# We should be able to enable this in the future
+%global want_d 0
+ 
+# Can't do anything with java with all the build deps in modules
+%global want_java 0
+ 
+%if 0%{?want_java} == 0
+%global java_configure --without-java
+%else
+%global java_configure --with-java
+%endif
+ 
+# Thrift's Ruby support depends on Mongrel.  Since Mongrel is
+# deprecated in Fedora, we can't support Ruby bindings for Thrift
+# unless and until Thrift is patched to use a different HTTP server.
+%if 0%{?have_mongrel} == 0
+%global ruby_configure --without-ruby
+%global with_ruby 0
+%else
+%global ruby_configure --with-ruby
+%global want_ruby 1
+%endif
+ 
+# Thrift's Erlang support depends on the JSX library, which is not
+# currently available in Fedora.
+ 
+%if 0%{?have_jsx} == 0
+%global erlang_configure --without-erlang
+%global want_erlang 0
+%else
+%global erlang_configure --with-erlang
+%global want_erlang 1
+%endif
+ 
+# PHP appears broken in Thrift 0.9.1
+%global want_php 0
+ 
+%if 0%{?want_php} == 0
+%global php_langname %{nil}
+%global php_configure --without-php
+%else
+%global php_langname PHP,\ 
+%global php_configure --with-php
+%endif
+ 
+# Thrift's GO support doesn't build under Fedora
+%global want_golang 0
+%global golang_configure --without-go
+ 
+# Thrift's Lua support has not yet been worked on
+%global want_lua 0
+%global lua_configure --without-lua
+ 
+# NOTE: thrift versions their libraries by package version, so each version
+# change is a SONAME change and dependencies need to be rebuilt
+Summary: Software framework for cross-language services development
+Name:    thrift
+Version: 0.15.0
+Release: 1%{?dist}
```

Comment:
Can you set the release to 5, since we are taking the Fedora spec version  0.15.0-4